src/screens/Ticketing/Ticket/ValidityLine.tsx

```diff
@@ -57,6 +58,46 @@ const ValidityLine: React.FC<{
   }
 };
 
+const LineWithVerticalBars = ({
+  status,
+  validityPercent = 100,
+}: {
+  status: 'valid' | 'reserving';
```

Comment:
Hmm. You mean like creating an enum, and using `Exclude<Status, Status.Expired>` to reuse the type.

Maybe it is better just having a `backgroundColor` or `progressBarStyle` parameter for setting the line background color, instead of passing the status?